src/test/utils/EigenLayerUnitTestSetup.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/token/ERC20/presets/ERC20PresetFixedSupply.sol";
+import "src/contracts/strategies/StrategyBase.sol";
+import "src/test/mocks/StrategyManagerMock.sol";
+import "src/test/mocks/DelegationManagerMock.sol";
+import "src/test/mocks/SlasherMock.sol";
+import "src/test/mocks/EigenPodManagerMock.sol";
+import "src/test/mocks/StakeRegistryMock.sol";
+import "src/test/utils/Utils.sol";
+import "src/test/utils/EigenLayerUnitTestBase.sol";
+
+abstract contract EigenLayerUnitTestSetup is EigenLayerUnitTestBase, Utils {
+    // Declare Mocks
+    StrategyManagerMock strategyManagerMock;
+    DelegationManagerMock public delegationManagerMock;
+    SlasherMock public slasherMock;
+    EigenPodManagerMock public eigenPodManagerMock;
+    StakeRegistryMock stakeRegistryMock;
+    // strategies
+    IERC20 public weth;
```

Comment:
These strategies can be removed, as can the imports. Also it looks like the contract doesn't need to inherit from Utils anymore

Reply:
Yup check my commit here https://github.com/Layr-Labs/eigenlayer-contracts/pull/293/commits/3a00d9367128a0bede8b648924f279b7523404f8
I removed the Utils and made the StrategyManagerUnit.t.sol inherit it as its the only contract using the `deployNewStrategy()` function

Reply:
Actually just moved that function into `StrategyManagerUnit.t.sol` and deleted `utils/Utils.sol` altogether. Its not being used anywhere else